src/integration_tests/schemas/e2e_schema_integration_test.py

```diff
@@ -59,6 +59,24 @@ def test_schema_e2e(self):
         response_as_json = test_schema_get_response.json()
         assert len(response_as_json) > 0
 
+        for survey_id in test_list_survey_id:
+            schema_post_response = session.post(
+                f"{config.API_URL}/v1/schema?survey_id={survey_id}",
+                json=test_schema,
+                headers=headers,
+            )
+
+        list_unique_survey_id = test_list_survey_id
+        list_unique_survey_id.insert(0, {test_survey_id})
+
+        set_list_unique_survey_id_response = session.get(
+            f"{config.API_URL}/v1/survey_list",
+            headers=headers,
+        )
+
+        assert set_list_unique_survey_id_response.status_code == 200
+        assert set_list_unique_survey_id_response.json() == list_unique_survey_id
```

Comment:
If it is just for testing the endpoint it might be worth creating an integration test that publishes schemas with different survey ids and a couple with the same then call the endpoint to check only the unique survey ids are published.

Reply:
@Mohammed1804 Thanks for the review, just refactored this and created a new integration test to check the list of unique survey IDs.